pkgs/tools/graphics/xcolor/default.nix

```diff
@@ -11,7 +11,8 @@ rustPlatform.buildRustPackage rec {
     sha256 = "sha256-rHqK05dN5lrvDNbRCWGghI7KJwWzNCuRDEThEeMzmio=";
   };
 
-  cargoSha256 = "sha256-lHOT/P1Sh1b53EkPIQM3l9Tozdqh60qlUDdjthj32jM=";
+  cargoPatches = [ ./update-cargo-lock.diff  ];
```

Comment:
Can you also send this patch upstream? It would be good to inform upstream that their cargo lock files are not up-to-date to prevent this issue in future.

Reply:
https://github.com/Soft/xcolor/pull/22